net.go

```diff
@@ -232,6 +240,8 @@ func Listen(laddr ma.Multiaddr) (Listener, error) {
 	case "utp":
 		// nl, err = utp.Listen(lnet, lnaddr)
 		return nil, fmt.Errorf("utp is currently broken")
+	case "udt", "udt4", "udt6":
```

Comment:
i think that multiaddr might yield a 4 or a 6? i could have sworn i saw it. (could be crazy)